### PR TITLE
Added quiz about golden images, Packer, and immutable infra

### PR DESCRIPTION
**Pull request by cfculler**
Added quiz and slightly reformatted the markdown for the section to match the rest of the bootcamp

### CODE REVIEW

File: src/quizzes/chapter-3/3.1/immutable-infra-and-packer-quiz.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+# Which statement is true about immutable infrastructure as opposed to mutable infrastructure?
+
+1. [x] Immutable infrastructure has less risk and is less complex
+1. [ ] Immutable infrastructure has more risk but is less complex
+	> It might prove beneficial to go watch the [What is Mutable vs. Immutable Infrastructure](https://www.hashicorp.com/resources/what-is-mutable-vs-immutable-infrastructure/) video again
+1. [ ] Immutable infrastructure has less risk but is more complex
+	> It might prove beneficial to go watch the [What is Mutable vs. Immutable Infrastructure](https://www.hashicorp.com/resources/what-is-mutable-vs-immutable-infrastructure/) video again
+1. [ ] Immutable infrastructure has more risk and is more complex
+	> It might prove beneficial to go watch the [What is Mutable vs. Immutable Infrastructure](https://www.hashicorp.com/resources/what-is-mutable-vs-immutable-infrastructure/) video again
```
Comment (cfculler) [inline src/quizzes/chapter-3/3.1/immutable-infra-and-packer-quiz.js:15]: While the term "less complex" is rather broad here, it was referenced directly in the video, and refers to removing the possibility of random "in between" versions that mutable infra poses due to certain upgrades working and others failing. With immutable, you either get version 1 or 2, not some weird in between one

Comment (jburns24) [reply src/quizzes/chapter-3/3.1/immutable-infra-and-packer-quiz.js:15]: I appreciate this comment

File: docs/3-virtual-machines-containers/3.1-golden-images.md [hunk 2]
```diff
@@ -68,7 +68,7 @@ For this exercise, you will create a Golden Image by manually installing and con
 
 Golden images can be a critical part of a very important strategy for managing software environments known as **Immutable Infrastructure**. We will cover Immutable Infrastructure in more depth throughout the DevOps Bootcamp. For now, watch this video of HashiCorp co-founder and CTO Armon Dadgar explaining [What is Mutable vs. Immutable Infrastructure](https://www.hashicorp.com/resources/what-is-mutable-vs-immutable-infrastructure/).
 
-## Automation
+## Automation and Packer
```
Comment (jburns24) [inline docs/3-virtual-machines-containers/3.1-golden-images.md:71]: Probably should just call this section `Packer`

**Review by jburns24 — APPROVED** [review]

File: docs/3-virtual-machines-containers/3.1-golden-images.md (hunk 2 above)
Comment (cfculler) [inline docs/3-virtual-machines-containers/3.1-golden-images.md:71]:
```suggestion
## Packer
```

**Pull request merged**